charts/k8s-service/values.yaml

```diff
@@ -114,6 +115,16 @@ readinessProbe: {}
 #   runAsUser: 2000
 securityContext: {}
 
+# podSecurityContext holds pod-level security attributes and common container settings. Optional: Defaults to empty. See type description for default values of each field
```

Comment:
This part of the sentence doesn't make sense, as there is no type description here. Can you either annotate with a link to the type descritpion, or remove it?

> See type description for default values of each field ...